test/quantization/test_quantize_script.py

```diff
@@ -1465,9 +1580,7 @@ def forward(self, x):
                 FileCheck().check_not("quantized::add_scalar") \
                            .run(m.graph)
 
-    @unittest.skipUnless('fbgemm' in torch.backends.quantized.supported_engines,
-                         " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
-                         " with instruction set support avx2 or newer.")
+    @skipIfNoFBGEMM
```

Comment:
Should we be able to run this test for qnnpack if fbgemm is not present?

Reply:
maybe this will work, but we can enable it in a separate diff

Reply:
Sounds good